docker/build_install_pythons.sh

```diff
@@ -7,17 +7,24 @@ apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6A755776
 apt-get update
 apt-get install -y wget
 PIP_ROOT_URL="https://bootstrap.pypa.io"
-for pyver in 2.7 3.6; do
+for pyver in 2.7; do
     pybin=python$pyver
     apt install -y ${pybin} ${pybin}-dev ${pybin}-tk
     wget $PIP_ROOT_URL/pip/$pyver/get-pip.py -O get-pip-$pyver.py
     get_pip_fname="get-pip-${pyver}.py"
     ${pybin} ${get_pip_fname}
 done
+for pyver in 3.6; do
+    pybin=python$pyver
+    apt install -y ${pybin} ${pybin}-dev ${pybin}-distutils ${pybin}-tk
+    wget $PIP_ROOT_URL/pip/$pyver/get-pip.py -O get-pip-$pyver.py
+    get_pip_fname="get-pip-${pyver}.py"
+    ${pybin} ${get_pip_fname}
+done
 wget $PIP_ROOT_URL/get-pip.py
 for pyver in 3.7; do
     pybin=python$pyver
-    apt install -y ${pybin} ${pybin}-dev ${pybin}-tk
+    apt install -y ${pybin} ${pybin}-dev ${pybin}-distutils ${pybin}-tk
```

Comment:
Could this be folded into the 3.8 3.9. 3.10 stanza below?

Reply:
You're right, thanks for catching. I've updated the commit